components/Form/EditSensorForm.jsx

```diff
@@ -0,0 +1,130 @@
+'use client'
+
+import { useState } from 'react'
+import { useRouter } from 'next/navigation'
+
+import { Input } from '@/components/ui/Input/Input'
+import { Button } from '@/components/ui/Button/Button'
+import { Select } from '@/components/ui/Select/Select'
+
+import { PageHeader } from '@/components/Page/PageHeader'
+import { Typography } from '@/components/ui/Typography'
+import { StationType } from '@prisma/client'
+import { useTranslations as getTranslations } from 'next-intl'
+
+export default function EditSensorForm({
+  id,
+  identifier,
+  type,
+  nbr_measures,
+  station_id,
+  records,
+  files,
+}) {
+  const [newIdentifier, setNewIdentifier] = useState(identifier)
```

Comment:
A best practice when you have a lot of field is to have only one `useState` to manage them. You can create a formData here representing the sensor. Example :
```
const [formData, setFormData] = useState(sensor)

  const handleChange = (inputName) => (evt) => {
    setFormData({
      ...formData,
      [inputName]: evt.target.value,
    })
  }
```
To see this example working you can look at [signup page](https://github.com/astrolabe-expeditions/data-platform/blob/bd6239008a2ad0f42ae39576ad6f8530c6728f3b/app/%5Blocale%5D/auth/signup/page.jsx)